tests/agent_tests/agent_tests.cc

```diff
@@ -78,3 +79,93 @@ TEST_P(AgentTests, SetThenGetEconParam) {
   EXPECT_EQ(data, -1.0);
 }
 
+TEST_P(AgentTests, PV_A_and_F) {
+  double present_value = agent_->PV(10, 0.05, 10000.0, 100.0);
+  double pv_hand_calc = 6911.306028;
+  EXPECT_NEAR(present_value, pv_hand_calc, 1e-3);
+}
+
+TEST_P(AgentTests, PV_only_F) {
```

Comment:
You can probably combine a few different tests into a single function:
* PV_only_F
* PV_only_A
* PV_A_and_F

Could also try a few different values of i and n.  So `test_PV` could be:
* i=0, n=0, F=1, A=0
* i>0, n=1, F=1, A=0
* i>0, n>1, F=1, A=0
* i>0, n>1, F>1, A=0
* i>0, n>1, F=0, A=1
* i>0, n>1, F=0, A>1
* i>0, n>1, F>1, A>1
Etc

It's also valid to test them against a computed response rather than a hand calculated response. E.g.
`pv(2,0.01,1,0) == pow(1.01,2)`